src/VoyagerCommand.php

```diff
@@ -62,18 +65,31 @@ public function fire()
         Artisan::call('vendor:publish', ['--provider' => 'Intervention\Image\ImageServiceProviderLaravel5']);
 
         $this->info("Migrating the database tables into your application");
-        Artisan::call('migrate');
+        Artisan::call('migrate:refresh');
```

Comment:
`migrate:refresh` does not sound like that good of an idea.
How about people installing Voyager for existing projects?